src/iterative_telemetry/__init__.py

```diff
@@ -190,10 +190,20 @@ def send(
         impl(payload)
 
     def _send_daemon(self, payload):
-        cmd = (
-            f"import requests;requests.post('{self.url}',"
-            f"params={{'token':'{self.token}'}},json={payload})"
-        )
+        cmd = f"""
+import requests, logging
+
+logger = logging.getLogger("iterative-telemetry")
+try:
+    requests.post(
+        '{self.url}',
+        params={{'token':'{self.token}'}},
+        json={payload},
+        timeout=10
+    )
+except Exception as e:
+    logger.debug(f'Telemetry request failed: {{str(e)}}')
```

Comment:
Can you even see this logging message? `logger.debug` is not shown by default, and you need some handlers setup.

Reply:
Right now, we don't. Also, I think that is the goal, not to show the stacktrace of telemetry to the users because they don't want it.

Reply:
Then why are we adding this block instead of suppressing it?
 
```suggestion
    pass
```

Reply:
Having a log may enable us to debug in development effort in case we want to setup some handlers. 
But I am okay with supressing it as well.